test/markup/elixir/strings.txt

```diff
@@ -1,4 +1,8 @@
-a = """test"""
-b = '''test'''
```

Comment:
I see how these are better examples (that you added), but if the previous were valid we should leave them to serve as regression tests... one can still use a multi-line string for a single line - that is valid grammar, yes?

Reply:
> that is valid grammar, yes?

No, those were syntax errors in Elixir. The only allowed characters after the opening `'''`/`"""` is optional whitespace, and then a newline is required.

My assumption is that it's not a big deal if the syntax highlighting colors something that doesn't compile, but the test examples should be valid Elixir code, right?

Reply:
> No, those were syntax errors in Elixir. 

Whoa.   I stand corrected. ;-)

> My assumption is that it's not a big deal if the syntax highlighting colors something that doesn't compile 

Right, it's not our job to detect invalid code.

> but the test examples should be valid Elixir code, right?

For the most part, yes.  In this particular case this indeed makes sense.  I just see people "swap" test cases often when they should be adding new cases instead, [not removing old ones].  Thought this was perhaps yet another case of that. :)